src/SearchField/SearchField.scss

```diff
@@ -12,6 +12,7 @@
 
     &.has-focus {
         box-shadow: 0 0 0 $search-border-width $search-border-color-interaction;
+        border: $search-border-focus-width double $search-border-focus-color;
```

Comment:
@PKulkoRaccoonGang When the user focused in the `SearchField` ([link](https://deploy-preview-1202--paragon-openedx.netlify.app/components/searchfield/)) and gets the extra outline (presumably from `double` here), it adds extra height to the element and causes it move around and change it's height. This may cause all its sibling UI elements to jump around too when the user focused, which isn't great.

How can we get this outline effect without the moving around or changing height?

Reply:
We use pseudo-elements, thanks